### ISSUE TITLE
Calendar: vertical 

### ISSUE THREAD

**Issue by kotecky** [runbox/runbox7#577]
The size of an hour slot is rather high. It is a compromise between how small/big the text of the displayed meetings need to be, and how big a portion of the day it allows to display. 

I would argue that the user should be given an overview of the full day at once, in the user's "core hours", defined in the prefs. This is surely very personal. For me from 8am to 9pm.

If details of a meeting need to be displayed, they should be made available on touch via a 'popup'.

**Comment by tadzik** [comment]
> It is a compromise between how small/big the text of the displayed meetings need to be

Actually, it seems like we can have the cake and eat it too: since events now have a minimal height anyway (well: will, in the next release :)), we could make each hour be a lot smaller and still have everything fit on the screen. Clicking each of these still brings up their details: 
![2020-05-21-153745_1003x861_scrot](https://user-images.githubusercontent.com/86378/82564513-2dcd5180-9b79-11ea-9f63-9d1c7270c2a2.png)

Core hours is another feature we could implement, but I'd say that even in the default view it's more useful to be able to see the entire day at a glance (perhaps zoomable). What do you think?

**Comment by kotecky** [comment]
This screenshot is a view in a computer browser, A responsive app should be able to accommodate both, serve cakes to mobiles and computers, small and large screens at the same time. On a mobile, it would be something like this (simulated on Win10 FF): 

![runbox_mobile-phone-simulation](https://user-images.githubusercontent.com/2297444/82567218-5ce5c200-9b7d-11ea-972f-16b443b84ec1.png)

For all events in a glance other apps use the "agenda" view that displays all events of a day, without the gaps, which is a need different from the day/week/month view.

**Comment by tadzik** [comment]
> This screenshot is a view in a computer browser, A responsive app should be able to accommodate both, serve cakes to mobiles and computers

Good point, I agree. I tried the same layout on a (simulated by Firefox) iPhone screen...

![2020-05-21-161739_456x740_scrot](https://user-images.githubusercontent.com/86378/82568174-b8647f80-9b7e-11ea-901a-7a9e20ea2e45.png)

...and while it looks okay-ish it's not exactly responsive yet. I'll keep this in mind when fixing this – ideally it should just take as much space as the screen allows, but not more.

The agenda/overview mode is available (since https://github.com/runbox/runbox7/pull/336) – though as we discovered in https://github.com/runbox/runbox7/issues/575 it's a bit hard to reach on mobile currently once you get stuck in daily/weekly view.

I'll see to creating a meta-task of Calendar responsiveness issues – thanks for bringing these up.

**Comment by kotecky** [comment]
> iPhone screen
On a _real_ mobile phone (FF on a Sony Xperia X Compact) the bottom of the display is at 7am.
